components/admin_console/filter/filter.tsx

```diff
@@ -188,6 +188,7 @@ class Filter extends React.PureComponent<Props, State> {
                 ref={this.filterRef}
             >
                 <button
+                    // eslint-disable-next-line @typescript-eslint/naming-convention
```

Comment:
I missed this earlier, but this might be something to keep in mind when discussing our CSS naming conventions in the future

Reply:
Yeah, came in after-the-fact. Will probably be other conflicts like this as other PRs get merged.

Should I go ahead and loosen `memberLike` in eslint?